dart/lib/src/sentry_tracer.dart

```diff
@@ -16,9 +17,12 @@ class SentryTracer extends ISentrySpan {
   final Map<String, dynamic> _extra = {};
   Timer? _autoFinishAfterTimer;
   var _finishStatus = SentryTracerFinishStatus.notFinishing();
+  var _trimEnd = false;
 
   SentryTracer(SentryTransactionContext transactionContext, this._hub,
-      {bool waitForChildren = false, Duration? autoFinishAfter}) {
+      {bool waitForChildren = false,
+      Duration? autoFinishAfter,
+      bool trimEnd = false}) {
```

Comment:
```suggestion
      bool trimEnd = false,}) {
```

Requires identation.